rebar.config

```diff
@@ -1,5 +1,5 @@
 {erl_opts, [debug_info]}.
-{deps, [{opencensus, "~> 0.7.0"},
+{deps, [{opencensus, ">= 0.7 < 0.9"},
```

Comment:
This is not supported by rebar3. If this "worked" it means it likely has a bug that results in it ignoring either the 0.7 or the 0.9.

Reply:
Ah, gotcha. In that case, how could we express this?

Reply:
You can't. rebar3 doesn't try to resolve a version that matches all declared constraints, it takes a match for the first occurrence of the dep it encounters.

So if your project using `opencensus_datadog` also requests `opencensus 0.8.0` it doesn't matter what version the `opencensus_datadog` dep has for opencensus.

Reply:
It appears that `mix` is a bit more strict than `rebar3`.
```
Failed to use "opencensus" (version 0.8.0) because
  deps/opencensus_plug/mix.exs requires >= 0.6.0 and <= 0.8.0
  oc_datadog (version 0.2.0) requires ~> 0.7.0
  opencensus_elixir (version 0.2.0) requires ~> 0.8.0
```

Reply:
Argh, yea, wasn't thinking about mix. I think you have to use `override: true` at the top level for now. I think we just have to remove `opencensus` as a dep from these since it was only needed for being able to declare the behaviour.

Reply:
@tsloughter @sneako I have done such version limitation due to fact that I have proposed a lot of changes to the `opencensus` itself, and if census-instrumentation/opencensus-erlang#136 land then it will break if someone do not lock on correct versions.